### ISSUE TITLE
NMM 62.21 restoring backup leads to no mods being recognized in FO4 

### ISSUE THREAD

**Issue by Rimmusin** [Nexus-Mods/NMM-Test-Builds#140]
I just restored a backup but when loading FO4 it shows that all of my mods are missing for the current savegame. After reinstalling some mods manually those are actually working again, but automatically restored mods are not.

**Comment by DuskDweller** [comment]
Can you see the restored mod files in the game's Data folder?

**Comment by Rimmusin** [comment]
no, they are not in the game folder. to fix it have to, at least thats what i'm doing, uninstall them all and then reinstall. then they work again. i do get a message to overwrite from NMM though

**Comment by KunoMochi** [comment]
Rimmusin, are the overwrites only for mod files in the Virtual Install folder?

**Comment by DuskDweller** [comment]
Which virtual install mod are you currently using, default or multi-hd ?

**Comment by Rimmusin** [comment]
@KunoMochi yes they are. 
@DuskDweller sorry but I have no idea. I just used 'tools' - 'backup and restore'- 'create mod installation backup'. At the same time (i had no idea tbh) steam was updating FO4. After restarting FO4 trying to see if NMM created any errors i got a CTD strait from the menu, actually when still in the intro movie. I disabled all mods in NMM. then decided against it since i wanted to see which mod caused the CTD by manually disabling just one mod instead of all (No ctd when starting FO4 without a mod) so I figures the easiest way would be by tools, restore, restore backup. this seems to work until I started FO4. then I noticed that none of my way too many mods were loaded. they did get a green checkmark however (this all in .6221). I disabled them all again and am now, one by one, deinstalling and reinstalling them. So tell me..is it default of multi-hd?

**Comment by Rimmusin** [comment]
I figures...me smeegol

**Comment by DuskDweller** [comment]
Was this fixed ?

**Comment by KunoMochi** [comment]
I think I may be experiencing the same issue here: https://github.com/Nexus-Mods/NMM-Test-Builds/issues/177

**Comment by Rimmusin** [comment]
I've been reluctant to try tbh (sorry) as I had to manually uninstall and install all my mods again..maybe if i feel brave i'll try tomorrow..

**Comment by Rimmusin** [comment]
I tried (see also #177) but got this
![image](https://cloud.githubusercontent.com/assets/20307514/17261097/6c278c50-55d4-11e6-99c4-a19703f8f6a0.png)
